state/migration_export_test.go

```diff
@@ -1146,9 +1147,7 @@ func (s *MigrationExportSuite) newResource(c *gc.C, appName, name string, revisi
 func (s *MigrationExportSuite) TestRemoteApplications(c *gc.C) {
 	// NOTE: the key 'c#<name>' isn't overly useful for someone looking
```

Comment:
Ah sorry, my mistake. We use c# since metawatcher requires a single char global key and 'a' and 'r' are taken. We should look into that.